gibbon-compiler/src/Gibbon/L0/Typecheck.hs

```diff
@@ -34,13 +34,24 @@ newtype TcM a = TcM (ExceptT Doc PassM a)
 instance MonadFail TcM where
   fail = error
 
-runTcM :: TcM a -> PassM (Either Doc a)
+runTcM ::   TcM a -> PassM (Either Doc a)
```

Comment:
there are *many* space-only edits in type signatures in this file that don't make sense to me. Could we revert them?